src/DatePicker/Calendar.js

```diff
@@ -98,7 +100,11 @@ class Calendar extends Component {
       return false;
     }
 
-    return this.refs.calendar.isSelectedDateDisabled();
+    if (this.calendarRefs.calendar) {
```

Comment:
Most probably not, but I like to check for refs just to be safe and have a fallback, as I've run into errors before. Most probably just got to do with hot-reloading though...

I could remove it if you like?